corehq/apps/export/views.py

```diff
@@ -710,8 +713,12 @@ def _process_filters_and_specs(self, in_data):
         """Returns a the export filters and a list of JSON export specs
         """
         filter_form_data, export_specs = self._get_form_data_and_specs(in_data)
+        import re
+        regex = re.compile('(emw=){1}([^&]*)(&){0,1}')
+        matches = regex.findall(filter_form_data['emw'])
```

Comment:
I'm not sure exactly what comes through as `filter_form_data`, but my guess is a string that looks like a URL querystring?  If that's the case, you could look in to how django parses those to generate the `request.GET` dictionary thing.  I bet there's some function from django you could import to break these up more intuitively.